apps/store/src/components/ProductReviews/ProductAverageRating.tsx

```diff
@@ -44,7 +44,7 @@ export const ProductAverageRating = () => {
       </Head>
 
       <div className={wrapper}>
-        <CertifiedIcon className={certifiedIcon} size="1rem" />
+        <CertifiedIcon className={certifiedIcon} size="1.150rem" />
```

Comment:
nit: seems like a very specific size :) ideally all icons should be using the same sized bounding box
```suggestion
        <CertifiedIcon className={certifiedIcon} size="1.15rem" />
```